kayenta-signalfx/src/main/java/com/netflix/kayenta/signalfx/config/SignalFxManagedAccount.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Nike, inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.kayenta.signalfx.config;
+
+import com.netflix.kayenta.retrofit.config.RemoteService;
+import com.netflix.kayenta.security.AccountCredentials;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Data
+public class SignalFxManagedAccount {
+
+  @NotNull
+  private String name;
+
+  private String accessToken;
+
+  @NotNull
+  private RemoteService endpoint = new RemoteService().setBaseUrl("https://stream.signalfx.com");
```

Comment:
Guessing this url can never be something else?

Reply:
I wasn't seeing any evidence that it could be something different.
With there ingest APIs though there are different URI's.

With this, though it is just setting a default value correct? I know Sean in the Halyard PR was allowing this to be overridden.

Reply:
We could just hard code the API URI as a constant and remove it as a config option from this and Halyard until there is an actual use case for it.

Thoughts?

Reply:
If there is no utility in overriding it, I'd also lean towards hardcoding it as a constant and not making it configurable via halyard. If a use case presents itself, we can add the support.